playbooks/common/openshift-cluster/upgrades/rpm_upgrade.yml

```diff
@@ -25,6 +25,17 @@
   register: result
   until: result | success
 
+ name: Upgrade SDN package
+  package:
+    name: "{{ openshift.common.service_type }}-sdn-ovs{{ openshift_pkg_version }}"
+    state: present
+  when:
+    - component == "master"
+    - not openshift.common.is_atomic | bool
+    - openshift_use_openshift_sdn | default(true) | bool
+  register: result
+  until: result | success
```

Comment:
I think we need to remove https://github.com/openshift/openshift-ansible/pull/7653/files#diff-5344c00ce631e3249535f443dd403318R18

Reply:
Or rather, fine some way to omit it from the package list when upgrading. If the SDN is in use we need to make sure that we upgrade to the correct version of the SDN because if we don't specify it in the same transaction which would upgrade the node package because the yum dependency solver will pull in the atomic-openshift-sdn-3.9 package when updating the rest of the packages to 3.8, and then in turn it would upgrade all packages to 3.9 at that point in time.